tests/baselines/reference/typeGuardsWithInstanceOf.errors.txt

```diff
@@ -37,7 +41,13 @@ tests/cases/conformance/expressions/typeGuards/typeGuardsWithInstanceOf.ts(7,20)
         }
         v // Validator & Partial<OnChanges> via subtype reduction
         if (v.onChanges) {
+              ~~~~~~~~~
+!!! error TS2339: Property 'onChanges' does not exist on type 'C | (Validator & Partial<OnChanges>)'.

```

Comment:
this looks like a regression. I reported this case back then, because the effect of `instaceof` affected code outside of the `if` statement